client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfiguration.java

```diff
@@ -128,11 +128,6 @@ default void check() {
             checkArgument(maxNumRetries() == 0, "If meshProxy is configured then maxNumRetries must be 0");
             checkArgument(uris().size() == 1, "If meshProxy is configured then uris must contain exactly 1 URI");
         }
-        if (nodeSelectionStrategy().equals(NodeSelectionStrategy.ROUND_ROBIN)) {
-            checkArgument(
-                    !failedUrlCooldown().isNegative() && !failedUrlCooldown().isZero(),
```

Comment:
Thoughts on keeping the non-negative enforcement?

Reply:
Done